.circleci/Readme.md

```diff
@@ -0,0 +1,14 @@
+We use [circle-ci](https://circleci.com/gh/samtools/hts-specs) to build the pdfs and highlight the differences for each pull-request. This is done at several steps.
+
+1. Circle-ci runs `make` and copies the pdfs and log files to an "artifacts" directory for that build. 
+2. Circle-ci runs `make diffs` on the tex files that correspond to tex files that changed since the "merge base" (defined as the most recent common ancestor of the HEAD and master branches)  
+3. Circle-ci was provided with a Token to a user (circle-ci-bot). This Token has been given permission to make comments on the github repository. 
```

Comment:
The convention is _README.md_.

Presumably you mean **hts-specs-bot** here.

What I was hoping you would put in this README is details of _how all this is set up_ — if you disappeared and the CircleCI integration broke down, what would the rest of us need to do to set it up again? What samtools-related account exists on CircleCI? Who has the keys to hts-specs-bot? Exactly what permissions does it have? What are the steps to set things up? etc.

However all this may become moot as the CircleCI integration might be superseded by GitHub Actions in the coming months…